src/WebJobs.Script/Utility.cs

```diff
@@ -33,6 +33,11 @@ public static string GetSubscriptionId()
             return null;
         }
 
+        public static bool IsDynamic()
+        {
+            return ScriptSettingsManager.Instance.GetSetting(EnvironmentSettingNames.AzureWebsiteSku) == ScriptConstants.DynamicSku;
```

Comment:
Hey @brettsam, is this how you did dynamic SKU detection for the timeout work? Just curious if there is an implementation that Matt can reuse here.